core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/stacks/StackV4RequestToStackConverter.java

```diff
@@ -139,8 +139,9 @@ public Stack convert(StackV4Request source) {
         stack.setInstanceGroups(convertInstanceGroups(source, stack));
         measure(() -> updateCluster(source, stack, workspace),
                 LOGGER, "Converted cluster and updated the stack in {} ms for stack {}", source.getName());
-        stack.getComponents().add(getTelemetryComponent(stack, source));
-
+        if (!isTemplate(source)) {
```

Comment:
can you move into the `else` branch? or inside to `convertAsStack(source, stack)`